alacritty_terminal/src/config/mod.rs

```diff
@@ -22,11 +22,12 @@ use serde::{self, de, Deserialize};
 use serde_yaml;
 
 use crate::ansi::CursorStyle;
-use crate::cli::Options;
 use crate::index::{Column, Line};
 use crate::input::{Action, Binding, KeyBinding, MouseBinding};
 use crate::term::color::Rgb;
 
+pub use self::options::Options;
```

Comment:
I'm personally not a huge fan of `pub use` I think that directory structures mostly exist for a good reason and imports should be done at the top of the file so the lengths doesn't play a huge role either.

So I think removing this and using `use crate::config::options::Options` give a better overview over what's actually going on and where to find the struct itself.

Reply:
I personally feel like `...::config::Options` is better than `...::config::options::Options`, I'm only really using the file to avoid bloating config/mod.rs. So in this case I don't see the reason for the nesting, but I agree with you in some cases.

Either way, I don't care much, and if you'd be happier with it as `config::options::Options` then I'm fine with that.

Reply:
I'd be much happier without re-exporting. That usually just creates a mess an inconsistencies in my experience. I believe we currently have a bunch of instances both using `crate::term::Cell` and `crate::term::cell::Cell` and I would like for that not to happen with this.

Reply:
Ok, so I guess my question is then, should I put it in config directly or leave it and change all the `use `statements. I think I'm leaning towards just moving it into `config`. Maybe this leads nicely to some unification we might be able to do there?

Reply:
I'd leave it in a separate module right now.

There's nothing wrong with `crate::config::options::Options`.